package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@suffolklitlab/alkiln",
-  "version": "5.4.0",
+  "version": "5.1.0-sources-7",
```

Should probably change this:

```suggestion
  "version": "5.4.1-sources-8",
```

---

I'm never sure how to handle this. `v5.1.0-sources-7` is the one that has been published. `v5.4.1-sources-8` doesn't exist, though on a surface level it more accurately describes where in commit history this now sits. I'll be publishing it under a new version anyway, so I think I'm going to keep what I have and resolve this.